mgmt/rest/metric.go

```diff
@@ -171,7 +174,11 @@ func (s *Server) getMetricsFromTree(w http.ResponseWriter, r *http.Request, para
 	respond(200, b, w)
 }
 
-func respondWithMetrics(host string, mets []core.CatalogedMetric, w http.ResponseWriter) {
+func respondWithMetrics(host string, mets []core.CatalogedMetric, w http.ResponseWriter, version string) {
+	if version == "v2" {
+		apiV2.RespondWithMetrics(host, mets, w)
+		return
+	}
```

Comment:
I don't think that level of partitioning is necessary yet. Maybe once we get a lot of V2 methods or even a V3 to some of these routes we could create a V1 package and then call the specified method from that package. I don't think we need that yet though.